files/system/usr/share/ublue-os/just/60-custom.just

```diff
@@ -559,3 +561,27 @@ audit-secureblue:
         done
     fi
 
+    GHNS_TEST_STRING="Ensuring KDE GHNS is disabled"
+    KDE_GLOBALS_FILE="/etc/xdg/kdeglobals"
+    if test -e $KDE_GLOBALS_FILE; then
```

Comment:
can't this still return true if ghns=false is present but commented out?

Reply:
It could; fixed on next commit.